lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -124,19 +147,30 @@ window.runLighthouseForConnection = function(connection, url, options, requested
   lighthouseIsRunning = true;
   updateBadgeUI(url);
 
-  // Run Lighthouse.
-  return Runner.run(connection, runOptions)
-    .then(result => {
-      lighthouseIsRunning = false;
-      updateBadgeUI();
-      filterOutArtifacts(result);
-      return result;
-    })
-    .catch(err => {
-      lighthouseIsRunning = false;
-      updateBadgeUI();
-      throw err;
-    });
+  return enableOtherChromeExtensions(false)
```

Comment:
needs to be in `runLighthouseInExtension` as devtools runs `runLighthouseForConnection` (through `runLighthouseInWorker`) as well

(`updateBadgeUI` probably should be too. It gates itself internally by returning if no `chrome.runtime`, but that could quickly get confusing if we had a bunch of noop functions in the devtools path)